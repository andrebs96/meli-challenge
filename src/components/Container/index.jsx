import styled from 'styled-components'

export const Container = styled.div`
  max-width: 120rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: 1rem;
  padding-right: 1rem;
  height: inherit;
`

export default Container
