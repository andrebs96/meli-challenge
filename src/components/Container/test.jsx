import React from 'react'
import { render, screen } from '@testing-library/react'

import Container from '.'

describe('<Container />', () => {
  it('should render the Container', () => {
    const { container } = render(
      <Container>
        <span>Container Test</span>
      </Container>
    )

    expect(screen.getByText(/Container Test/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
