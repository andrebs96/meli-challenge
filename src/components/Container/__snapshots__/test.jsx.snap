// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render the Container 1`] = `
.c0 {
  max-width: 120rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: 1rem;
  padding-right: 1rem;
  height: inherit;
}

<div
  class="c0"
>
  <span>
    Container Test
  </span>
</div>
`;
